docs/Deploying-Pega-on-AKS.md

```diff
@@ -509,6 +512,18 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
 
 3. Save the file.
 
+### Changes in Backend setting if tiers.service.tls.domain is `true`
+
+If the setting `tiers.service.tls.enabled` is set to `true` and the certificate in the keystore is issued by a valied CA authority, no additional steps is required.
+else if the certificate in the keystore is a self-signed or not issued by a valid certificate authority, the following additional configuration is required in Azure application gateway,
```

Comment:
If you set `tiers.service.tls.enabled` to `true` and the certificate in the keystore is issued by a valid CA authority, no additional steps is required.
If the certificate in the keystore is a self-signed or not issued by a valid certificate authority, you must include the following additional configuration in your Azure application gateway,